index.html

```diff
@@ -17,13 +17,13 @@
   <body>
     <header class="head">
       <div class="logo">
-        <p>TRACY MIRANJA</p>
-      </div>
-      <div class="menu">
-        <img src="./images/Union.png" />
+        <a href="#">TRACY MIRANJA</a>
       </div>
       <nav>
-        <ul>
+        <div class="menu">
+          <img src="./images/Union.png" />
+        </div>
+        <ul class="links">
```

Comment:
- [ ] As requested by the previous reviewer :recycle: , you should stick as much as possible to the Figma design. Always look at the requirements as something asked by a client. Kindly make should the margin, the padding , the fonts ,,, etc. are the same in your design as they are in the Figma template.

Your design 

![Screenshot from 2023-01-12 10-28-34](https://user-images.githubusercontent.com/80458199/212016329-5b8018ac-d64f-4022-beda-a7c7e2721db0.png)


Figma design


![Screenshot from 2023-01-12 10-30-01](https://user-images.githubusercontent.com/80458199/212016558-3fe83132-3c96-47c4-b6a9-caa644d126a8.png)